### PR TITLE
Added feature to enable sum of strings

### PR DESCRIPTION
**Pull request by AlexanderDitz**
Changes sum function:
def sum(a: Union[int, str], b: Union[int, str]) -> Union[int, str]:

# Description

*If the issue is on Github, simply link to it using the '#' symbol)*:
- resolves issue #2

# Type of change
- **feat**: A new feature
> - "feat: new theorist abc",

# Features (Optional)
- bulleted list of features, describing the code contributions in this pull request

# Remarks (Optional)
- any additional remarks for the code reviewer
- if your remarks or questions pertain to individual lines in the code, you may also include them as comments in the code review window

# Screenshots (Optional)
If your PR is related to UI change or anything visual, feel free to include screenshots to give the reviewer straightforward understandings on your changes.

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -103,4 +103,4 @@ def return_random_number() -> int:
     float
     '''
 
-    return np.random.randint(0, 100)
```
Comment (parthapratimkalita) [inline src/utils.py:106]: why, it is happning?

File: src/utils.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # TODO: make all functions work with strings as well
 # TODO: add a new cool calculator function
 
-def sum(a: int, b: int) -> int:
+def sum(a: Union[int, str], b: Union[int, str]) -> Union[int, str]:
```
Comment (parthapratimkalita) [inline src/utils.py:7]: They only asked to make a function for to add integers and float, not string.

Comment (Krishnendu-Bose) [reply src/utils.py:7]: Yes. I agree with this.

File: src/utils.py (hunk 1 above)
Comment (Krishnendu-Bose) [reply src/utils.py:106]: I think because the previous line was replaced with an entire new line which is an exact copy of the previous line.

**Review by parthapratimkalita — APPROVED** [review]

**Pull request merged**